src/components/Formik/FormikCheckboxInput/FormikCheckboxInput.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {
+  FC,
+  ChangeEvent,
+  FocusEvent,
+} from 'react';
+import CheckboxInput from '../../CheckboxInput/CheckboxInput';
+
+interface FormikCheckboxInputProps {
+  field: {
+    id: string;
+    label: string;
+    name: string;
+    onBlur: (event: FocusEvent<HTMLElement>) => void;
+    onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+    value?: boolean;
+  };
+  form: { [key: string]: any; }; // eslint-disable-line @typescript-eslint/no-explicit-any
+}
+
+const FormikCheckboxInput: FC<FormikCheckboxInputProps> = ({
+  field: {
+    id,
+    label,
+    name,
+    onBlur,
+    onChange,
+    value = false,
+  },
+  form: { touched, errors },
+  ...props
+}) => (
+  <CheckboxInput
+    id={id}
+    label={label}
+    error={touched[name] && errors[name]}
+    isChecked={value}
+    onBlur={onBlur}
+    onChange={onChange}
+    {...props}
```

Comment:
I'm not sure the two `...props` spreads are necessary at all.

Reply:
@juanfabrega

Reply:
Hey @RyanMacBern the component would not work properly otherwise. It is meant to pass our own component's (`<CheckboxInput>`) props down through the chain. They are merged with the formik field's props. The first line takes all of our extra props and assign's them to the variable `props`, and the second line passes them down. Without each of those statements, the process breaks down. 

perhaps it would be clearer to rename these as `rest` or `restProps` to make it clear that these are props in addition to the default formik field props. I'm good either way, but maybe that helps clarify the intent.